pisek/compile.py

```diff
@@ -47,12 +50,15 @@ def compile(self, filepath: str, dry_run: bool = True) -> Optional[str]:
     def valid_shebang(filepath: str) -> bool:
         """ Check if file has shebang and if the shebang is valid """
 
-        with open(filepath, "r") as f:
+        with io.open(filepath, "r", newline="\n") as f:
```

Comment:
Dle StackOverflow je `io.open` ekvivalentní builtinu `open`.
https://stackoverflow.com/questions/33891373/difference-between-io-open-vs-open-in-python

Navrhuji:
```suggestion
        with open(filepath, "r", newline="\n") as f:
```
a smazat import, pokud to uznáš za vhodné :)